<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PogadeTransformationView</class>
 <widget class="QWidget" name="PogadeTransformationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonTransform">
           <property name="text">
            <string>Transform</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeTransformations">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>IDT</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Line</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Transformation</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutCodeOld">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Original Code</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutCodeNew">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>New Code</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonTransform</sender>
   <signal>clicked()</signal>
   <receiver>PogadeTransformationView</receiver>
   <slot>applyTransform()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applyTransform()</slot>
 </slots>
</ui>
